WalletWasabi.Fluent/ViewModels/Wallets/Receive/ReceiveAddressViewModel.cs

```diff
@@ -14,16 +15,20 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Receive;
 [NavigationMetaData(Title = "Receive Address")]
 public partial class ReceiveAddressViewModel : RoutableViewModel
 {
+	private readonly CompositeDisposable _disposables = new();
+	private readonly IWalletModel _wallet;
+
 	public ReceiveAddressViewModel(UiContext uiContext, IWalletModel wallet, IAddress model, bool isAutoCopyEnabled)
 	{
+		_wallet = wallet;
 		UiContext = uiContext;
 		Model = model;
 		Address = model.Text;
 		Labels = model.Labels;
 		IsHardwareWallet = wallet.IsHardwareWallet();
 		IsAutoCopyEnabled = isAutoCopyEnabled;
 
-		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		SetupCancel(false, true, true);
```

Comment:
This change is not needed and IMO it was more readable.

```suggestion
		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true)
```